packages/mdc-ripple/util.js

```diff
@@ -15,13 +15,21 @@
  */
 
 /** @private {boolean|undefined} */
+let supportsCssVariables_;
```

Comment:
Yes document both. These were never documented, and I still don't understand what they are trying to accomplish. We added annotation comments for the compiler...but it wasn't done by a core member of the team...he was just trying to get the compiler to pass.

"supportsCssVariables" -> "whether or not the current runtime execution supports custom css properties"? or "whether or not the current component supports the var syntax"? or something else?

"supportsPassive" -> I mean, passive what? Passive ripple? Passive browser?